packages/engine/src/output/utils.rs

```diff
@@ -19,6 +17,7 @@ pub fn cleanup_experiment(experiment_id: &ExperimentId, exit_status: EngineExitS
     let frompy_files = glob::glob(&format!("{experiment_id}-frompy*"));
     let topy_files = glob::glob(&format!("{experiment_id}-topy*"));
```

Comment:
The engine shouldn't really care what python files are generated by the runners, asking the Python runner to clean up its garbage may be sensible.

Reply:
But the Python runner might have died right? 

I don't mind there being a free-standing function called `clean_up_socket_files` or something that lives within the Python runner module, but as we've talked about before this isn't for graceful exits.

Reply:
Yes, I don't want the python runner to do this, but the module should provide a method for cleaning it up.

Reply:
Added a function in the runner https://github.com/hashintel/hash/pull/494/commits/683389f6efb7d1e44382dc79b96998088a81d87a